tests/openvino/test_quantization.py

```diff
@@ -451,30 +482,30 @@ def test_ov_model_static_quantization_with_auto_dataset(
             ov_model = model_cls.from_pretrained(model_id, quantization_config=quantization_config)
             ov_model.save_pretrained(tmp_dir)
 
+            submodels = ov_model.ov_submodels.values()
+
             if model_cls == OVModelForSpeechSeq2Seq:
-                submodels = [ov_model.encoder.model, ov_model.decoder.model]
-                if ov_model.decoder_with_past is not None:
-                    submodels.append(ov_model.decoder_with_past.model)
-                else:
+                if ov_model.decoder_with_past is None:
                     expected_num_weight_nodes_per_model = expected_num_weight_nodes_per_model[:-1]
                     expected_fake_nodes_per_model = expected_fake_nodes_per_model[:-1]
 
                 input_features = torch.randn((1, 128, 3000), dtype=torch.float32)
                 ov_model.generate(input_features)
-            elif model_cls == OVModelForCausalLM:
-                submodels = [ov_model]
-
+            elif model_cls in (OVModelForCausalLM, OVModelForFeatureExtraction):
                 tokenizer = AutoTokenizer.from_pretrained(model_id)
                 if tokenizer.pad_token is None:
                     tokenizer.pad_token = tokenizer.eos_token
                 tokens = tokenizer("This is a sample input", return_tensors="pt")
-                outputs = ov_model(**tokens)
-                self.assertTrue("logits" in outputs)
-            elif any(
-                x == model_cls
-                for x in (OVStableDiffusionPipeline, OVStableDiffusionXLPipeline, OVLatentConsistencyModelPipeline)
+                ov_model(**tokens)
+            elif model_cls in (
+                OVStableDiffusionPipeline,
+                OVStableDiffusionXLPipeline,
+                OVLatentConsistencyModelPipeline,
             ):
-                submodels = ov_model.ov_submodels.values()
+                # TODO: add inference check
+                pass
```

Comment:
is this todo for another PR ?

Reply:
Yes

Reply:
why not now ? it's as simple as `ov_model(prompt="a text to image prompt")`

Reply:
Indeed, added :slightly_smiling_face: